apps/barista-design-system/src/shared/services/toc.service.ts

```diff
@@ -76,27 +77,31 @@ export class BaTocService implements OnDestroy {
     this.tocItems = this._refractorTocItems(headlines, docId);
     this.tocList.next(this.tocItems);
 
-    this._zone.runOutsideAngular(() => {
-      // start the scroll spy
-      this._scrollSpyInfo = this._scrollSpyService.spyOn(headlines);
-    });
-    this._activeItemSubscription = this._scrollSpyInfo!.active.subscribe(
-      scrollItem => {
-        if (scrollItem) {
-          for (const tocItem of this.tocItems) {
-            const scrollItemId = scrollItem.element.getAttribute('id');
-            if (tocItem.headlineId === scrollItemId) {
-              this.activeItems.next([tocItem]);
-            }
-            for (const tocSubItem of tocItem.subheadlines) {
-              if (tocSubItem.headlineId === scrollItemId) {
-                this.activeItems.next([tocItem, tocSubItem]);
+    // TODO: whole scroll spy logic has to be fixed! It was not refactored
+    // during the angular router refactoring
```

Comment:
Can you please create a follow up issue for that one, @lukasholzer ?

Reply:
done: #796